Gopkg.toml

```diff
@@ -36,10 +44,30 @@
   branch = "master"
   name = "github.com/giantswarm/microstorage"
 
+[[constraint]]
+  branch = "master"
+  name = "github.com/giantswarm/operatorkit"
+
 [[constraint]]
   name = "github.com/go-kit/kit"
   version = "0.6.0"
 
 [[constraint]]
   name = "github.com/spf13/viper"
   version = "1.0.0"
+
+[[override]]
+  name = "k8s.io/api"
+  version = "kubernetes-1.8.5"
+
+[[override]]
+  name = "k8s.io/apiextensions-apiserver"
+  version = "kubernetes-1.8.5"
+
+[[override]]
+  name = "k8s.io/apimachinery"
+  version = "kubernetes-1.8.5"
+
+[[override]]
+  name = "k8s.io/client-go"
+  version = "kubernetes-1.8.5"
```

Comment:
Are you sure those guys don't work with `[[constraint]]`? Have you tried the `required` trick?